superset/config.py

```diff
@@ -166,6 +166,17 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 NATIVE_FILTER_DEFAULT_ROW_LIMIT = 1000
 # max rows retrieved by filter select auto complete
 FILTER_SELECT_ROW_LIMIT = 10000
+
+# SupersetClient HTTP retry configuration
+# Controls retry behavior for all HTTP requests made through SupersetClient
+# This helps handle transient server errors (like 502 Bad Gateway) automatically
+SUPERSET_CLIENT_RETRY_ATTEMPTS = 3  # Maximum number of retry attempts
```

Comment:
could take either side of that debate, but would a nested object a better idea than multiple config keys here?

Reply:
Wouldn't a nested object be harder to configure via env vars using that refactoring you started?

Reply:
oh I think it might be better, though not sure yet, but was hoping for JSON schema + antd autoform magic